perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -55,7 +55,8 @@
                     'Storage region for object storage benchmark.')
 
 flags.DEFINE_enum('object_storage_scenario', 'all',
-                  ['all', 'cli', 'api_data', 'api_namespace'],
+                  ['all', 'cli', 'api_data', 'api_namespace',
+                   'api_multistream'],
```

Comment:
Add comments below in the flag help text to clarify the difference between api_data and api_multistream. (yes, I know the help text as it stands is missing api_data)